src/CreditsChecker.php

```diff
@@ -10,10 +10,12 @@
 
 class CreditsChecker
 {
+    private ClientFactory $clientFactory;
     private StorageApiClient $client;
 
-    public function __construct(StorageApiClient $client)
+    public function __construct(ClientFactory $clientFactory, StorageApiClient $client)
```

Comment:
Jako slyším to 🐰 🕳️ , ale když tady děláme BC break, nemohlib bysme akceptovat StorageToken a vysrat se na to ošmatlávání SAPI v týhle třídě?

Reply:
Asi radeji udelam pristi tyden dalsi major release nez se do toho ted poustet :sweat_smile: